spec/models/candidates/registrations/placement_request_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+describe Candidates::Registrations::PlacementRequest, type: :model do
+  it { is_expected.to have_db_column(:date_start).of_type :date }
```

Comment:
Is it worth ensuring the not null columns have `not null` set by using `.with_options(null: false)`?